### PR TITLE
Allow search and filter simultaneously, update collection when search returns no results.

### PR DESCRIPTION
**Pull request by aidanfeay**

### CODE REVIEW

**Comment by ryanmark** [comment]
Why did you move the search box and remove the clear button?

**Comment by aidanfeay** [comment]
I combined the forms so they can submit/clear together. They couldn't be used simultaneously because the submission of one would only pass its own params and reset the other.

File: appjs/templates/project_list.ejs [hunk 1]
```diff
@@ -62,6 +48,16 @@
             <% }) %>
             </select>
           </div>
+            <label class="sr-only" for="search">Search query</label>
+            <div class="btn-group">
+              <input type="text" class="form-control"
+                     name="search" id="search" placeholder="Search"
+                     <% if(query.search) { %>value="<%=query.search %>"<% } %>>
+            <% if(query.search) { %>
```
Comment (ryanmark) [inline appjs/templates/project_list.ejs]: Doesn't the clear link only show up if you've done a search? How do you clear if you've applied a theme filter?

**Comment by aidanfeay** [comment]
Combined the conditionals to fix that.

**Comment by ryanmark** [comment]
:+1:

**Pull request merged**